scripts/createFromTemplate.ps1

```diff
@@ -187,6 +187,11 @@ Get-ChildItem -Force -File -Recurse * | ForEach-Object {
                 ForEach-Object {
                     $_ -replace "__home__",$env:HOME
                 } | Set-Content $a
+                ( Get-Content $a ) |
+                ForEach-Object {
+                    # Get just the last string of the path, which is the name of the folder of the template
+                    $_ -replace "__templateFolder__",$(Split-Path -Path $templateFolder -Leaf)
```

Comment:
We have the `$template` variable that is the "name" of the template that can be used here